rov/sensors/imu/Adafruit_BNO055/BNO055.py

```diff
@@ -1,694 +0,0 @@
-# Adafruit BNO055 Absolute Orientation Sensor Library
```

Comment:
This file got removed and instead added to the requirements file, since it's in pip